rust/README.md

```diff
@@ -9,44 +9,24 @@ natively from Rust.
 Internally, it relies on [`bindgen`](https://docs.rs/bindgen/) and
 [`libloading`](https://docs.rs/libloading/).
 
-## Compiling the model
-
-The Rust wrapper does not currently have any functionality to compile Stan models.
-Compiled shared libraries need to be built manually using `make` or with the Julia
-or Python bindings.
-
-For safety reasons all Stan models need to be installed with `STAN_THREADS=true`.
```

Comment:
I think this and the later note about how an error will be thrown at runtime if it is not are still worth saying, if a user is using a library compiled from outside of their Rust program.

Reply:
Fair point, the reason i removed it is that if the model compile method is invoked, then it will recompile the model if it wasn't compiled with stan threads previously, but it's true that there is always a risk of someone not calling the compile method